### DISCUSSION TITLE
Proposal for cursor interaction with follow circles

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#20524] [Ideas]
This is how slider dragging looks now.
![изображение](https://user-images.githubusercontent.com/53872073/192915633-d893a293-0b94-48a4-a500-80ca9a0dcfe4.png)
There are too many borders. While playing with this skin, i got an idea that it could be cool if cursor border (more exactly, white-red circle around it) would also serve as slider follow-circle. I mean, instead of pink circle that pops out of slider ball, quickly move+resize cursor border to where follow circle border is now. The cursor itself (white glowing dot) is following player's input as usual, so you can move it inside cursor/followcircle border as usual. If you break a slider, i.e. moving your cursor out of cursor/followcircle borders, return their size and position back where they should be (around the cursor). I'm talking only about animations here (cursor border should fade to pink while moving to "following" state to make final look be as now it is).

I am not sure that it will look good, won't be too distracting and that i worded it well, but anyways.

**Comment by peppy** [comment]
Sounds like an interesting proposal but I don't know how it would feel in practice.
